src/app/pages/AddAsset.tsx

```diff
@@ -131,7 +131,7 @@ const Form: FC = () => {
         throw new NotMatchingStandardError('Failed when detecting token standard');
       }
 
-      await assertGetBalance(tezos, contract, tokenStandard, tokenId);
+      if (tokenStandard === 'fa2') await assertFa2TokenDeployed(tezos, contract, tokenId);
```

Comment:
wrong name, FA 2 token will be deployed but he just could have not such ID in storage